File: pgm/ABC_Dart/ABC_393_A.dart
// import 'dart:collection';
import 'dart:io';
// import 'dart:math';

void main() {
  final [String S1, String S2] = stdin.readLineSync()!.split(' ').toList();
  const int one = 1;
  const int two = 2;
  const int three = 3;
  const int four = 4;
  const String sick = 'sick';
  const String fine = 'fine';
  if (S1 == sick && S2 == sick) {
    print(one);
  } else if (S1 == sick && S2 == fine) {
    print(two);
  } else if (S1 == fine && S2 == sick) {
    print(three);
  } else {
    print(four);
  }
  exit(0);
}

File: pgm/ABC_PHP/ABC_393_A.php
<?php
$stream = fopen('php://stdin', 'r');
$line = fgets($stream);
fclose($stream);
const sick = 'sick';
const fine = 'fine';
const space = ' ';
const one = 1;
const two = 2;
const three = 3;
const four = 4;
const zero = 0;
$S = array_map('strval', explode(space, trim($line)));
echo $S[zero] === sick && $S[one] === sick
    ? one
    : (
        $S[zero] === sick && $S[one] === fine
        ? two
        : (
            $S[zero] === fine && $S[one] === sick
            ? three
            : four
        )
    ) . PHP_EOL;
exit(0);
